daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -18,6 +18,25 @@
 FAIL=0
 FAILURES=""
 
+function wait_for_connectivity {
+  if [[ $(cat /etc/centos-release) =~ "CentOS release 6" ]]; then
```

Comment:
CentOS and RHEL 6 are identical in this realm. If you ran into this in one you can run into it in the other as well.

Reply:
You're right!

This `wait_for_connectivity` is a surgical fix for the CentOS6 OVF integration test. The test has been failing for awhile, and I want to make sure that we get it passing so that it doesn't hide other errors.

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/c4dd645bf723d7a934a719a865ee8f6f7afce6bb/gce_ovf_import_tests/test_suites/ovf_import_test_suite.go#L113 .

Zooming out, @hopkiw is working on a fix that will address both CentOS6 and RHEL6 in https://github.com/GoogleCloudPlatform/compute-image-packages/blob/master/packages/google-compute-engine/src/etc/init/google-startup-scripts.conf

Reply:
Centos 6 tests were failing for regular image import, not only OVF import. Your fix will apply to both though so no action needed.